crates/hyperswitch_connectors/src/connectors/tsys.rs

```diff
@@ -40,11 +39,20 @@ use hyperswitch_interfaces::{
 };
 use transformers as tsys;
 
+use super::utils::convert_amount;
 use crate::{constants::headers, types::ResponseRouterData, utils};
+#[derive(Clone)]
+pub struct Tsys {
+    amount_converter: &'static (dyn AmountConvertor<Output = StringMajorUnit> + Sync),
+}
```

Comment:
Please use `StringMinorUnit` since we are accepting amount in minor unit which was being passed directly into the connector. It seems the link referred in the issue for documentation is incorrect, no need to refer it for implementation.

Reply:
done